arangod/Cluster/ClusterComm.cpp

```diff
@@ -332,13 +326,29 @@ void ClusterComm::cleanup() {
 ////////////////////////////////////////////////////////////////////////////////
 
 void ClusterComm::startBackgroundThread() {
-  _backgroundThread = new ClusterCommThread();
+  _roundRobin = 0;
```

Comment:
Can we initialize `_roundRobin` to 0 in the constructors instead?

Reply:
Apart from that, this method should be renamed to `startBackgroundThreads`, because it can now start multiple threads.

Reply:
done